test/components/TopNav/NavMenuTest.js

```diff
@@ -12,34 +13,57 @@ import { Link } from 'react-router-dom'
 
 function mockDispatch (object) { }
 
-
 function setup() {
-    let props = {
-        dispatch: mockDispatch
-    }
-  return createComponent(NavMenu, props)
+  return shallow(<NavMenu dispatch={mockDispatch} ticketIds={['1111', '2222', '3333']}/>)
 }
 
 describe('NavMenu', function test () {
-    beforeEach( () => {
-        this.output = setup()
-    })
+  beforeEach( () => {
+    this.wrapper = setup()
+  })
 
-    it('Should render an IconMenu with an icon button', () => {
-        expect(this.output.type).to.equal(IconMenu)
-        expect(this.output.props.iconButtonElement.type).to.equal(IconButton)
-    })
+  it('Should render an IconMenu with an icon button', () => {
+    expect(this.wrapper.type()).to.equal(IconMenu)
+    expect(this.wrapper.props().iconButtonElement.type).to.equal(IconButton)
+  })
 
-    it('Should render an Incident Search link', () => {
-        expect(this.output.props.children[0].type).to.equal(MenuItem)
-        expect(this.output.props.children[0].props.primaryText.type).to.equal(Link)
-        expect(this.output.props.children[0].props.primaryText.props.to).to.equal('/search')
-    })
+  it('Should render an Incident Search link', () => {
+    let link = this.wrapper.props().children[0]
+
+    expect(link.type).to.equal(MenuItem)
+    expect(link.props.primaryText.type).to.equal(Link)
+    expect(link.props.primaryText.props.to).to.equal('/search')
+  })
+
+  it('Should render a log out link', () => {
+    let link = this.wrapper.props().children[1]
+
+    expect(link.type).to.equal(MenuItem)
+    expect(link.props.primaryText.type).to.equal(Link)
+    expect(link.props.primaryText.props.to).to.equal('/')
+    expect(link.props.primaryText.props.onClick).to.exist
+  })
 
-    it('Should render a log out link', () => {
-        expect(this.output.props.children[1].type).to.equal(MenuItem)
-        expect(this.output.props.children[1].props.primaryText.type).to.equal(Link)
-        expect(this.output.props.children[1].props.primaryText.props.to).to.equal('/')
-        expect(this.output.props.children[1].props.primaryText.props.onClick).to.exist
+  it('Should render a log out link', () => {
```

Comment:
May have forgotten to update the description

Reply:
🤦🏻‍♀️